cadc-util/src/main/java/ca/nrc/cadc/auth/ServletPrincipalExtractor.java

```diff
@@ -164,6 +164,12 @@ public ServletPrincipalExtractor(final HttpServletRequest req)
             finally { }
         }
 
+        String authToken = request.getHeader("Authorization");
```

Comment:
I would declare a constant like AuthenticationUtil.HTTP_AUTHORIZATION_HEADER with this value and document it as being defined in HTTP/1.1